Wait, Linux isn’t succeeding 
```
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:136 (message):
    Command failed: /ci/myagent/_work/3/s/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/bin/cmake --build . --config Debug --target install -- -v
    Working Directory: /ci/myagent/_work/3/s/buildtrees/ppconsul/x64-linux-dbg
    See logs for more information:
      /ci/myagent/_work/3/s/buildtrees/ppconsul/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:91 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:24 (vcpkg_build_cmake)
  ports/ppconsul/portfile.cmake:17 (vcpkg_install_cmake)
  scripts/ports.cmake:94 (include)


Error: Building package ppconsul:x64-linux failed with: BUILD_FAILED
Elapsed time for package ppconsul:x64-linux: 7.38 s
```

--- a/ports/ppconsul/CONTROL
+++ b/ports/ppconsul/CONTROL
@@ -1,5 +1,5 @@
 Source: ppconsul
-Version: 0.4
+Version: 0.4-1
 Homepage: https://github.com/oliora/ppconsul
 Description: A C++ client library for Consul. Consul is a distributed tool for discovering and configuring services in your infrastructure.
-Build-Depends: boost-core, boost-variant, boost-optional, curl[openssl], json11
+Build-Depends: boost-core, boost-variant, boost-optional, boost-fusion, curl[openssl], json11

--- a/ports/ppconsul/portfile.cmake
+++ b/ports/ppconsul/portfile.cmake
@@ -13,6 +13,7 @@ file(REMOVE_RECURSE ${SOURCE_PATH}/ext/catch)
 
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
 )
 vcpkg_install_cmake()
 

